src/buildercore/bootstrap.py

```diff
@@ -256,6 +272,14 @@ def update_sqs_stack(stackname, **kwargs):
         # stack has/had queues that may need to be created/destroyed
         setup_sqs(stackname, current_context_sqs, current_context['project']['aws']['region'])
 
+def update_terraform_stack(stackname, **kwargs):
+    if 'FASTLY_API_KEY' not in os.environ:
+        raise ConfigurationException("You must provide a FASTLY_API_KEY environment variable to provision Fastly resources")
+    
+    working_dir = join(TERRAFORM_DIR, stackname)
+    t = Terraform(working_dir=working_dir)
+    t.apply(input=False, capture_output=False, raise_on_error=True)
```

Comment:
`apply()` calls require a confirmation from the user typing `yes`. There is a lot of potential around using `plan` and `apply` in two separate phases, but it's still unclear how to integrate that for now.